helpers/email/email.py

```diff
@@ -1,26 +1,49 @@
 from .email_setup import SendEmail
 from config import Config
 from flask import render_template
+from api.room.models import Room as RoomModel
 
 
-def send_email_notification(admin_email, location_name, user_name):
-    # send the email
-    recipients = [admin_email]
+class EmailNotification:
```

Comment:
Great Job on the PR so far @ThaDeveloper. Please could you provide documentation for the class and method that you created in this file

Reply:
@Makwe-O This has been handled